### ISSUE TITLE
Source code blocks rendering problems

### ISSUE THREAD

**Issue by mats852** [beejjorgensen/bgc#102]
Hi !

There seems to be a problem when rendering the font-size of some elements in the code blocks on iOS browsers:

<div style="display: flex;">
<img src="https://user-images.githubusercontent.com/6082025/177072736-bc9914be-c8e8-435b-be15-ea02445e0eda.PNG" width="49.5%" />

<img src="https://user-images.githubusercontent.com/6082025/177072847-77fb40f6-56ee-431c-8b98-c4bc34a3c3b6.PNG" width="49.5%" />
</div>

The demonstrated solution above was to remove `display: inline-block;` from the code block span elements.

```diff
pre > code.sourceCode > span {
-  display: inline-block;
  line-height: 1.25;
}
```

Another issue is with their backgrounds:

<div style="display: flex;">
<img src="https://user-images.githubusercontent.com/6082025/177074695-276e5325-b88d-4617-9f97-e983b5c024aa.png" width="49.5%" />

<img src="https://user-images.githubusercontent.com/6082025/177074909-e412dcde-7f50-4375-aaea-6edd949d81d8.png" width="49.5%" />
</div>

Which can be fixed using `overflow: scroll` for the `pre` elements using the `.sourceCode` class but seems to be working when we update this code globally.

```diff
.sourceCode {
-  overflow: visible;
+  overflow: scroll;
}
```

Thanks! Great read so far! :)

**Comment by beejjorgensen** [comment]
Ok, I made these changes--but I have to admit the first one freaks me out a little since I don't see why it would change the font size, and I'm not sure what regressions it might introduce. But it seems to be working.

For the second one, I made a different fix because I didn't want scroll bars on all the desktop code snippets when they weren't needed. I just moved the background out to the containing div.

Uploading now.
